utils/requester.py

```diff
@@ -158,3 +162,22 @@ def render(self):
             if status_code >= 500:
                 bug_seq.append(new_seq)
         return [valid_seq, bug_seq]
+
+    def simple_fuzz_render(self, fuzzer):
+        valid_seq = []
+        bug_seq = []
+        requests_str_list = []
+        if "ddos_fuzzer" in fuzzer:
```

Comment:
Since we know all fuzzers in this variable is valid, we can do something like:
```py
for fuzzer in fuzzers:
  requests_str_list.extend(Fengine.fuzz(POSSIBLE_FUZZERS[fuzzer], self.last_req_original, self.datatypes))
```

and get rid of the two if statements